lib/JSON/Validator.pm

```diff
@@ -85,7 +92,7 @@ sub bundle {
       my $ref  = ref $from;
 
       if ($ref eq 'HASH' and my $tied = tied %$from) {
-        return $from if $tied->fqn =~ m!^\#!;
+        return $from if $tied->fqn =~ m!^$this_fqn\#!;
```

Comment:
What happens if `$this_fqn` is empty string?

Reply:
That's the current behaviour. See the line it's replacing?

Reply:
I must have been sleeping. Thank you.